### ISSUE TITLE
Loss of attachment of posterior dural sac resulting in flattening of cervical cord

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10928]
**Preferred term label:**
Loss of attachment of posterior dural sac resulting in flattening of cervical cord

**Synonyms**

**Definition (free text, please give PubMed ID)**
Anterior shift of dural sac seen on MRI cervical spine flexion view when compared to neutral view

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/books/NBK499913/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

I think what is observed is the following -- thought?

Label: Forward displacement of posterior cervical dura mater on neck flexion
Definition: Flexion MRI (imaging with neutral and flexed - 30-40 degrees neck position) reveals an anterior shift of the cervical dural sac from the lamina in the region of C4 to C7.
Comment: This finding is characteristic of Monomelic amyotrophy, or Hirayama disease, is a rare motor neuron disorder characterized by progressive weakness and atrophy in the distal upper extremities, typically beginning during adolescence. The condition involves asymmetric muscle atrophy in the hand and forearm, sparing the brachioradialis muscle, and is attributed to increased laxity in the dura mater, which causes the spinal cord to shift forward during neck flexion. Increased laxity of the dura mater from its superior anchors on the dorsal surfaces of C2 and C3, harbingering forward cord movement during cervical flexion. Increased laxity allows the spinal cord to shift forward, leading to repeated subclinical trauma and microischemia in the anterior horn during cervical flexion. Trauma and microischemia eventually lead to myelopathy and degeneration, as evidenced by asymmetric lower cervical cord thinning within the C7 and C8.

PMID:29763088, PMID:36475209

**Comment by MickeySegal** [comment]
I would change:
Monomelic amyotrophy, or Hirayama disease, is a rare motor neuron
to 
Monomelic amyotrophy (Hirayama disease) is a rare motor neuron

Otherwise it looks good to me but let me run it by the neurologist who contributed this to make sure.

**Comment by MickeySegal** [comment]
The neurologist says this is good.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001191
- label: Forward displacement of posterior cervical dura mater on neck flexion
- parents: [Brain imaging abnormality (HP:0410263)](https://hpo.jax.org/app/browse/term/HP:0410263)
- pmid: PMID:29763088|PMID:36475209
- def: Flexion MRI (imaging with neutral and flexed about 30-40 degrees neck position) reveals an anterior shift of the cervical dural sac from the lamina in the region of C4 to C7.
- comment: This finding is characteristic of Monomelic amyotrophy (Hirayama disease) is a rare motor neuron disorder characterized by progressive weakness and atrophy in the distal upper extremities, typically beginning during adolescence. The condition involves asymmetric muscle atrophy in the hand and forearm, sparing the brachioradialis muscle, and is attributed to increased laxity in the dura mater, which causes the spinal cord to shift forward during neck flexion. Increased laxity of the dura mater from its superior anchors on the dorsal surfaces of C2 and C3, harbingering forward cord movement during cervical flexion. Increased laxity allows the spinal cord to shift forward, leading to repeated subclinical trauma and microischemia in the anterior horn during cervical flexion. Trauma and microischemia eventually lead to myelopathy and degeneration, as evidenced by asymmetric lower cervical cord thinning within the C7 and C8.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10928
- synonyms:
